docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,663 @@
+
+Customizing the admin is as simple as creating a file in the `./admin` folder of your app. You need to make sure the file you want to customize is at the same location in your `./admin` folder as it is in the `strapi-admin` package. For a reference you can look at the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin).
+
+For the beta there are quite a lot of changes that were made to the admin. If you did customize things you will have to verifiy if the file still exists in the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin) or find its new location.
+
+### Example
+
+You can modify the logo of the app by creating a file `./admin/src/assets/images/logo-strapi.png` to replace the admin logo.
+
+You can do the same with any file in the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin) of the admin panel
+
+## Migrating your database
+
+Starting from `beta` we are adding a new model `Administrator` to replace the `users-permissions` plugin `User` model for connecting to the `Strapi admin panel` panel.
+
+The goal is to emphasize the difference between an `Administrator` of the `Strapi admin panel` and a `User` of the application you are building with Strapi. Thus making the `users-permissions` plugin focused on providing an authentication and authorization service to your application.
+
+This means that, from now on you will have a new `strapi_administrator` collection in your database. Since this new table will be empty on startup, you will be asked to create a new administrator.
+
+You can either create this new administrator and manually recreate all your administrators or, create a migration script that will copy your `users` with `administrator` role to the `strapi_administrator` collection.
+
+In the short term the `Administrator` model will not be customisable. If you need have created relations between your models and the `User` model (e.g: author) to link to the administrator you will have to keep these users in both collections.
+
+## Running your migrated project
+
+To run your migrated project you will now need to run `strapi develop` or `npm run develop` to run the project in watch mode (e.g auto reloading on content-type creation).
+
+To run strapi without watch mode then run `strapi start` or `npm run start`
+
+Finally if you want to run your project in different envrionments use `NODE_ENV=env npm run start`
+
+## Updating Deployments
+
+When deploying a Strapi project you will have to make sure the `build` script is ran. (e.g `npm run build`)
+
+Depending on your deployment environment you might have to run the `build` script with `NODE_ENV=production npm run build` if the envrionment doesn't include it by default.
+
+For example `heroku` will automatically run the `build` script for you and set `NODE_ENV=production` before running the scripts so you won't have to think about it.
+
+## Conclusion
```

Comment:
Maybe we should remove this header?

Reply:
@davidkartuzinski Can you create an awesome conclusion for us ? <3

Reply:
See below @alexandrebodin